ipapython/install/common.py

```diff
@@ -89,13 +89,24 @@ class Interactive(core.Configurable):
 
 
 class Continuous(core.Configurable):
+    def __init__(self, **kwargs):
+        super(Continuous, self).__init__(**kwargs)
+        self.exceptions_handled = 0
+
     def _handle_execute_exception(self, exc_info):
         try:
             super(Continuous, self)._handle_execute_exception(exc_info)
+        except SystemExit:
+            pass
         except BaseException as e:
             logger.debug("%s", traceback.format_exc())
             if isinstance(e, Exception):
                 logger.error("%s", e)
+            self.exceptions_handled = 1
+
+    def run(self):
+        super(Continuous, self).run()
+        return self.exceptions_handled
```

Comment:
That's fine for developers but won't values not in (0,1) confuse users?

Reply:
I see your point. Sadly there is near zero documentation on how these classes are expected to work so I'm trying to tread gently. It is my assumption based on the class name Continuous and the fact that exceptions aren't fatal that this is expected to soldier in in most cases of failures so returning that number seems a reasonable thing to return. I'm not necessarily opposed to returning 0 or 1.